hw/ip/rom_ctrl/data/rom_ctrl.hjson

```diff
@@ -31,6 +31,17 @@
       randcount: "128",
       randtype:  "data"
     }
+
+    { name:      "SecDisableScrambling",
+      type:      "bit",
+      desc:      '''
+        Disable scrambling and checking in rom_ctrl, turning the block into a simple
```

Comment:
Nit: You don't need to wrap the line here.

Reply:
Maybe, you could add also a sentence here that this is for emulation purposes on small FPGAs only, but not for a real design.

Reply:
Good idea!